docs/local.rst

```diff
@@ -10,7 +10,8 @@ PynamoDB has been tested with two DynamoDB compatible servers, `DynamoDB Local <
 and `dynalite <https://github.com/mhart/dynalite>`_.
 
 All you need to do is set the ``host`` attribute on your ``Model``'s ``Meta`` class to the hostname and port
-that your server is listening on.
+that your server is listening on. If you use `rate_limited_scan` on your models with DymamoDB, you must also
+set `allow_rate_limited_scan_without_consumed_capacity` to `True` in the `settings file <settings.rst>`_.
```

Comment:
Nit: it may be better to link to the specific setting on the page rather than the page as a whole.

Also note that you'll need to use double backticks for code blocks in rst